pom.xml

```diff
@@ -8,7 +8,7 @@
 	</parent>
 
 	<artifactId>hp-application-automation-tools-plugin</artifactId>
-	<version>5.1-SNAPSHOT</version>
+	<version>5.0</version>

```

Comment:
- Please don't change the POM unless you need to add a new dependency.
- This is the wrong version number.